examples/jsx/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "hello-nuxt-jsx",
+  "name": "nuxt-jsx",
   "dependencies": {
-    "nuxt": "latest"
+    "nuxt": "next"
```

Comment:
I believe we should keep `latest` since we won't have to update examples when 1.0 will be released.

Reply:
I just want to make it useable for now and change it when 1.0 is going to release.
But now I think you're right, keep  consistency is more important, I will make the change. 😸

Reply:
For information, I always try the examples with: `bin/nuxt examples/hello-world/` :)